### DISCUSSION TITLE
Unable to find the AggregatorV3Interface.sol

### DISCUSSION THREAD

**Discussion by Harshvardhantomar** [Cyfrin/foundry-full-course-cu#1606] [Q&A] [answered]
I am unable to find the Aggregator solidity file in smartcontractkit of ChainLink. Could anyone explain and help me, how do I find that file?

**Comment by EngrPips** [comment]
Hey @Harshvardhantomar , are you working on remix or locally on your machine ?, Also someone recently had similar issue https://github.com/Cyfrin/foundry-full-course-f23/discussions/1604#discussion-6509023 you might check the discussion out to see if that helps

**Reply by Harshvardhantomar** [reply]
I am working on Remix. Let me check the issue and update you.

**Reply by Harshvardhantomar** [reply, marked as answer]
Got it. I was not deploying the contract on Injected Metamask. Therefore I was getting no results.

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. keep crushing it!
